# DP: Description: #257840: locales: Please add the eu_FR locale
# DP: Related bugs: #257840
# DP: Dpatch author: GOTO Masanori <gotom@debian.org>
# DP: Patch author: Christian Perrier <bubulle@debian.org>
# DP: Upstream status: Not submitted
# DP: Status Details: sending upstream
# DP: Date: 2004-07-31

---
 localedata/SUPPORTED          |    3 +
 localedata/locales/eu_FR      |   83 ++++++++++++++++++++++++++++++++++++++++++
 localedata/locales/eu_FR@euro |   83 ++++++++++++++++++++++++++++++++++++++++++
 3 files changed, 169 insertions(+)

--- /dev/null
+++ b/localedata/locales/eu_FR
@@ -0,0 +1,83 @@
+comment_char %
+escape_char  /
+%
+% Basque Language Locale for France
+% Language: eu
+% Territory: FR
+% Revision: 1.0
+% Date: 2004-06-24
+% Users: general
+% Charset: ISO-8859-1
+% Distribution and use is free, also
+% for commercial purposes.
+
+LC_IDENTIFICATION
+title      "Basque locale for France"
+source     "Christian Perrier and the Debian Project"
+address    ""
+contact    "Christian Perrier"
+email      "bubulle@debian.org"
+tel        ""
+fax        ""
+language   "Basque"
+territory  "France"
+revision   "1.0"
+date       "2004-06-24"
+%
+category  "eu_FR:2000";LC_IDENTIFICATION
+category  "eu_FR:2000";LC_CTYPE
+category  "eu_FR:2000";LC_COLLATE
+category  "eu_FR:2000";LC_TIME
+category  "eu_FR:2000";LC_NUMERIC
+category  "eu_FR:2000";LC_MONETARY
+category  "eu_FR:2000";LC_MESSAGES
+category  "eu_FR:2000";LC_PAPER
+category  "eu_FR:2000";LC_NAME
+category  "eu_FR:2000";LC_ADDRESS
+category  "eu_FR:2000";LC_TELEPHONE
+
+END LC_IDENTIFICATION
+
+LC_COLLATE
+copy "iso14651_t1"
+END LC_COLLATE
+
+LC_CTYPE
+copy "i18n"
+END LC_CTYPE
+
+LC_MESSAGES
+copy "eu_ES"
+END LC_MESSAGES
+
+LC_MONETARY
+copy "fr_FR"
+END LC_MONETARY
+
+LC_NUMERIC
+copy "fr_FR"
+END LC_NUMERIC
+
+LC_TIME
+copy "eu_ES"
+END LC_TIME
+
+LC_PAPER
+copy "fr_FR"
+END LC_PAPER
+
+LC_TELEPHONE
+copy "fr_FR"
+END LC_TELEPHONE
+
+LC_MEASUREMENT
+copy "fr_FR"
+END LC_MEASUREMENT
+
+LC_NAME
+copy "eu_ES"
+END LC_NAME
+
+LC_ADDRESS
+copy "fr_FR"
+END LC_ADDRESS
--- a/localedata/SUPPORTED
+++ b/localedata/SUPPORTED
@@ -191,6 +191,9 @@
 eu_ES.UTF-8/UTF-8 \
 eu_ES/ISO-8859-1 \
 eu_ES@euro/ISO-8859-15 \
+eu_FR.UTF-8/UTF-8 \
+eu_FR/ISO-8859-1 \
+eu_FR@euro/ISO-8859-15 \
 fa_IR/UTF-8 \
 ff_SN/UTF-8 \
 fi_FI.UTF-8/UTF-8 \
--- /dev/null
+++ b/localedata/locales/eu_FR@euro
@@ -0,0 +1,83 @@
+comment_char %
+escape_char  /
+%
+% Basque Language Locale for France
+% Language: eu
+% Territory: FR
+% Revision: 1.0
+% Date: 2004-06-24
+% Users: general
+% Charset: ISO-8859-15
+% Distribution and use is free, also
+% for commercial purposes.
+
+LC_IDENTIFICATION
+title      "Basque locale for France"
+source     "Christian Perrier and the Debian Project"
+address    ""
+contact    "Christian Perrier"
+email      "bubulle@debian.org"
+tel        ""
+fax        ""
+language   "Basque"
+territory  "France"
+revision   "1.0"
+date       "2004-06-24"
+%
+category  "eu_FR:2000";LC_IDENTIFICATION
+category  "eu_FR:2000";LC_CTYPE
+category  "eu_FR:2000";LC_COLLATE
+category  "eu_FR:2000";LC_TIME
+category  "eu_FR:2000";LC_NUMERIC
+category  "eu_FR:2000";LC_MONETARY
+category  "eu_FR:2000";LC_MESSAGES
+category  "eu_FR:2000";LC_PAPER
+category  "eu_FR:2000";LC_NAME
+category  "eu_FR:2000";LC_ADDRESS
+category  "eu_FR:2000";LC_TELEPHONE
+
+END LC_IDENTIFICATION
+
+LC_COLLATE
+copy "iso14651_t1"
+END LC_COLLATE
+
+LC_CTYPE
+copy "i18n"
+END LC_CTYPE
+
+LC_MESSAGES
+copy "eu_ES"
+END LC_MESSAGES
+
+LC_MONETARY
+copy "fr_FR"
+END LC_MONETARY
+
+LC_NUMERIC
+copy "fr_FR"
+END LC_NUMERIC
+
+LC_TIME
+copy "eu_ES"
+END LC_TIME
+
+LC_PAPER
+copy "fr_FR"
+END LC_PAPER
+
+LC_TELEPHONE
+copy "fr_FR"
+END LC_TELEPHONE
+
+LC_MEASUREMENT
+copy "fr_FR"
+END LC_MEASUREMENT
+
+LC_NAME
+copy "eu_ES"
+END LC_NAME
+
+LC_ADDRESS
+copy "fr_FR"
+END LC_ADDRESS
